felix/bpf/ut/nat_encap_test.go

```diff
@@ -106,7 +106,6 @@ func checkVxlan(pktR gopacket.Packet) gopacket.Packet {
 	udpL := pktR.Layer(layers.LayerTypeUDP)
 	Expect(udpL).NotTo(BeNil())
 	udpR := udpL.(*layers.UDP)
-	Expect(udpR.SrcPort).To(Equal(layers.UDPPort(testVxlanPort)))
```

Comment:
Would be nice to have some check; e.g. could check that two packets with same source port hash to same but different source ports hash differently?